src/systemd-rdma-settings.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+SERV=`echo $1 | awk -F@ '{print $1}'`
+FILE="/usr/lib/systemd/system/"$SERV"@.service"
+ARG_DIR="/run/systemd/system/"$1""
+NAME="$1"
+FLAG=false
+
+add_config(){
+	if [ -f "/run/systemd/system/ceph-osd.target.wants/"$NAME"" ]; then
+		sudo rm -rf "/run/systemd/system/ceph-osd.target.wants/"$NAME""
+	fi
+
+	for CONF in "$@"
+	do
+		if [ ! -f $CONF ]; then
+			touch $CONF
+			FLAG=true
+			cat $FILE > $CONF
+		fi
+		if ! grep -q "[Service]" $CONF ; then
+			echo "[Service]"| sudo tee -a $CONF
+			FLAG=true
+		fi
+		if grep -q "PrivateDevices=yes" $CONF; then
+			sudo sed -i 's/PrivateDevices=yes/PrivateDevices=no/g' $CONF
```

Comment:
is it possible that the services are started in parallel? so the ceph.conf is edited by multiple processes? if that's the case, we need a lock file in `/var/run`.